authzed/api/v1/experimental_service.proto

```diff
@@ -296,36 +296,33 @@ message ExpPermission {
 
 message ExperimentalComputablePermissionsRequest {
   Consistency consistency = 1;
-  repeated ExpRelationReference relations = 2;
+  string definition_name = 2;
+  string relation_name = 3;
 
   // optional_definition_name_match is a prefix that is matched against the definition name(s)
   // for the permissions returned.
   // If not specified, will be ignored.
-  string optional_definition_name_filter = 3;
+  string optional_definition_name_filter = 4;
 }
 
 // ExpRelationReference is a reference to a relation in the schema.
 message ExpRelationReference {
   string definition_name = 1;
   string relation_name = 2;
-}
-
-// ExpPermissionReference is a reference to a permission in the schema.
-message ExpPermissionReference {
-  string definition_name = 1;
-  string permission_name = 2;
+  bool is_permission = 3;
```

Comment:
Why a bool instead of `ExpRelationReference` and `ExpPermissionReference`?

Reply:
Because otherwise we'd need two distinct lists to be returned and this is easier to generate

Reply:
Do we treat permissions and relations as the same thing anywhere else in the API?
Whether we choose to do this or not just needs to be consistent across the entire API surface. We have to decide whether SpiceDB users need to know that permissions a form relation.